tools/scripts/find-broken-docs-links.py

```diff
@@ -1,10 +1,27 @@
+#!/usr/bin/env python3
+
+# ruff: noqa: S607 This scripts runs in a controlled environment.
```

Comment:
```suggestion
```

sorry for sharing the wrong tips to supress it - i recently came to realize this as well:

we can run `["/usr/bin/env", "git", ...]` to specifically refer to the command we want to run in the environment